### PR TITLE
Feature#37 feed design

### LINKED ISSUE #37: Feed Design
# REQUIREMENTS
- Similar to mockup

# DESIGN
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Still works
- [ ] Similar to mockup

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
What did you implement/Why did you implement this:
 - Add a design to our feed page.

  ## Screenshots 
Front-end
Screenshot from 2022-02-15 00-00-15
mockup design
Feed

### CODE REVIEW

File: app/views/profiles/index.html.erb [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
+          <h3 class="user__name"> <%= link_to current_user.name, profile_path(current_user.id) %> </h3>
+          <% skill = Skill.find_by(user_id: current_user.id ) %>
+          <% if !skill.nil? %>
+            <h4 class="user__skill" > <%= skill.name %> </h4>
+          <% else %>
+            <h4 class="user__skill"> <%= link_to "Add skill", new_skill_path(current_user.id) %> </h4>
+          <% end %>
+        <% end %>
+      </div>
+    </aside>
```
Comment (YilKanda) [inline app/views/profiles/index.html.erb]: What do you think about moving this code to an external file?

File: app/assets/stylesheets/components/_feed.scss [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      display: flex;
+      flex-direction: column;
+      gap: 0.5rem;
+    }
+
+    .line{
+      display: flex;
+      justify-content: space-between;
+      align-items: end;
+      width: 100%;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss]: sort this block

File: app/assets/stylesheets/components/_feed.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      display: flex;
+      gap: 1.5rem;
+    }
+
+    &__info{
+      width: 100%;
+      justify-content: center;
+      display: flex;
+      flex-direction: column;
+      gap: 0.5rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss]: Sort this block

File: app/assets/stylesheets/components/_feed.scss [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      color: $color_white;
+    }
+  }
+
+  .profile{
+    background-color: $feed_card;
+    display: grid;
+    padding: 2.5rem;
+    gap: 0.5rem;
+    margin-bottom: 1rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss]: Sort this block

File: app/assets/stylesheets/components/_feed.scss [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+    &__img{
+      width: 100%;
+    }
+
+    &__name{
+      margin: 0;
+      margin-top: 1rem;
+      text-align: center;
+    }
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss]: Please, in each of this file blocks add a space between your name class and  brackets

File: app/assets/stylesheets/components/_feed.scss [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    &__skill{
+      @extend .user__name;
+      margin-top: 0.5rem;
+    }
+
+    a{
+      margin-top: 0.5rem;
+      color: $color_black;
+      text-decoration: none;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss]: sort this block

File: app/assets/stylesheets/components/_feed.scss [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    .line{
+      display: flex;
+      justify-content: space-between;
+      align-items: end;
+      width: 100%;
+    }
+
+    &__img{
+      width: 8rem;
+      height: 8rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss]: sort it

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/profiles/index.html.erb (hunk 1 above)
Comment (xFerchoVJ) [reply app/views/profiles/index.html.erb]: I agree with @YilKanda

**Comment by xFerchoVJ** [comment]
Solve the feedback comments 👌🏻

File: app/views/profiles/index.html.erb (hunk 1 above)
Comment (JuanequeX) [reply app/views/profiles/index.html.erb]: x3

**Comment by DerekAyala** [comment]
@JuanequeX, @xFerchoVJ, @YilKanda could you give me another review?

File: app/assets/stylesheets/components/_feed.scss [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+    &__img {
+      width: 100%;
+    }
+
+    &__name {
+      margin-top: 1rem;
+      margin: 0;
+      text-align: center;
+    }
+    &__skill {
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_feed.scss:21]: Add a space here

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -12,12 +12,14 @@
...
 
   <body>
+    <div class="main">
     <% if user_signed_in? %>
       <%= render "shared/navbar"%>
       <p class="notice"><%= notice %></p>
       <p class="alert"><%= alert %></p>
     <% end %>
 
     <%= yield %>
```
Comment (JuanequeX) [inline app/views/layouts/application.html.erb]: please review this code identation

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by diegoOG09 — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**